practice_07.20.2019_multithreading/Task1and2_Matrix/FallingSymbols.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Threading;
+
+namespace practice_07._20._2019_multithreading.Task1and2_Matrix
+{
+    public class FallingSymbols
+    {
+        static readonly object locker = new object();
+
+        private readonly Random random;
+
+        private readonly int consoleWidth;
+
+        private readonly int consoleHeight;
+
+        public FallingSymbols()
+        {
+            this.random = new Random();
+            this.consoleWidth = 120;
+            this.consoleHeight = 50;
+            Console.SetWindowSize(this.consoleWidth, this.consoleHeight);
+            Console.CursorVisible = false;
+        }
+
+        public void ExecuteMatrix()
+        {
+            while (true)
+            {
+                Thread.Sleep(new Random().Next(20, 1000));
+                new Thread(new ThreadStart(this.PrintSymbols)).Start();
```

Comment:
Please use Task instead of Thread.
Do you remember that Thread creation is very costly?